app/Resources/TwigBundle/views/Exception/error500.html.twig

```diff
@@ -9,9 +9,8 @@
 
             <div class="error-content">
                 <h3><i class="fa fa-warning text-red"></i> T'as tout cassé !</h3>
-
                 <p>
-                    Si le problème persiste, tu nous envoyer un petit rapport d'erreur à <a href="mailto:dsi@N7consulting.fr">dsi@N7consulting.fr</a> <br>
+                    Si le problème persiste, tu peux nous envoyer un petit rapport d'erreur à <a href="mailto:{{ technical_to }}">{{ technical_to }}</a><br>
```

Comment:
Sending an email to their own IT is ok, I think they would have think to it.

I think we should add a notice to where an other source of support can be found, Github or by email. WDYT ?